.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: publish
+
+on:
+  release:
+    types: [published] # Trigger when release is created
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    env:
+      TEST_PYPI_TOKEN: ${{ secrets.TEST_PYPI_TOKEN }}
+      PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+    steps:
+    - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+    - name: Set up Python 3.7
+      uses: actions/setup-python@3105fb18c05ddd93efea5f9e0bef7a03a6e9e7df
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run:  |
+        pip install --upgrade pip
+        pip install --upgrade poetry
+    - name: Publish to Test-PyPI with Poetry
```

Comment:
we don't really need to publish to test-pypi, so we can remove this if we want.
I've tested, both steps should work when a Release is published

Reply:
probably skip this publish?